internal/k8s/extract.go

```diff
@@ -4,12 +4,12 @@ import (
 	"fmt"
 	"reflect"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
I thiiiiink this was a bug in goimports that got fixed?

Reply:
hmmmm...i just pulled a new version of goimports and it's still doing it?

Reply:
apparently it actually went in the other direction!

https://github.com/golang/go/issues/30051

maybe we should have ci verify w/ goimports instead of gofmt